CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
-
+## [1.4.7]
+Updates/enhancements:
+- Enable suppression of SLDS warnings in HTML files for mobile via Quick Fix actions. Any HTML content that is between `<!-- sldsValidatorIgnore -->` and `<!-- sldsValidatorAllow -->` will now be exempt from SLDS validation. Furthermore `<!-- sldsValidatorIgnoreNextLine -->` will exempt the next immediate line from SLDS validation. By adding a `<!-- sldsValidatorIgnore -->` at the top of an HTML file with no `<!-- sldsValidatorAllow -->` anywhere else in that file, you can exempt the entire content of the file from SLDS validation.
```

Comment:
@maliroteh-sf - this also change the behavior for CSS. Is that correct? Do we want to call that out here?

Reply:
@billyma  So I didn't change the CSS behavior. It still follows the old behavior. The only difference is that, now the user can choose to suppress a warning in a CSS file via a Quick Fix action (vs. manually having to type in the special token to disable a warning). Since we're still using the old suppression behavior in CSS files and since that behavior is different than what we use for HTML files, I thought not calling it out would be better, vs. calling it out and potentially causing confusion for the end user.

We really need to sit down and figure out how can we update CSS warning suppression to be the same as HTML files. In my opinion the suppression approach that we use in HTML files are simpler and powerful (suppressing an entire file / range of lines) but that would break backwards compatibility.

Reply:
Gotcha. Great, we can handle the CSS via a preference later. Thanks.